miflora/miflora_poller.py

```diff
@@ -7,144 +7,25 @@
 """
 
 from datetime import datetime, timedelta
-from threading import Lock, current_thread
-import re
-from subprocess import PIPE, Popen, TimeoutExpired
+from threading import Lock
 import logging
 import time
-import signal
-import os
-
-#from gattlib import GATTRequester
+from bluepy.btle import Peripheral, BTLEException
 
 MI_TEMPERATURE = "temperature"
 MI_LIGHT = "light"
 MI_MOISTURE = "moisture"
 MI_CONDUCTIVITY = "conductivity"
 MI_BATTERY = "battery"
 
+BYTEORDER = 'little'
+INVALID_DATA = b'\xaa\xbb\xcc\xdd\xee\xff\x99\x88wf\x00\x00\x00\x00\x00\x00'
+
```

Comment:
At least this is the only one that I've seen so far. Since there is no documentation of the device interface I can only guess.